### ISSUE TITLE
Fix geoModifiedStereographicAlaska

### ISSUE THREAD

**Issue by Fil** [d3/d3-geo-projection#174]
_TL;DR I think it’s safe to change the default to `.clipAngle(30)`, and would like to have your opinion on changing the default angle from 0 to `.angle(7.8)`._

geoModifiedStereographicAlaska has a few issues:

- by default it has clipAngle(25), which is not enough to reproduce figure 44 of [Snyder](https://books.google.se/books?id=nPdOAAAAMAAJ&pg=PA203&lpg=PA203#v=onepage&q&f=false). A clipAngle of 30 is enough (we can push that to 39° before having overlaps).

- The default angle() is 0, but the map is more balanced if its 160° meridian is almost vertical (making it vertical is not possible because it's not a straight line). This requires an angle of 7.3°. Or, probably even nicer, if the two spikes are aligned horizontally — that's with angle(7.8).

(I'm not sure we want to set default angles in the module — but it would make the projection look better in the list.)

- The coefficients are a bit different from those given in Snyder, it would be nice to know where these coefficients came from. (The impact on the map is not really visible.)

With changes in clipAngle and angle, the map becomes:

![alaska](https://user-images.githubusercontent.com/7001/59218688-1cd23d00-8bc1-11e9-9363-4880a1e6fd62.png)

**Comment by Fil** [comment]
Compare with the current version in d3-geo-projection:
![](https://raw.githubusercontent.com/d3/d3-geo-projection/master/img/modifiedStereographicAlaska.png)

And Jason's original version (which is based on the gs50 coefficients):
https://www.jasondavies.com/maps/modified-stereographic/alaska/

Explore the various possibilities at https://observablehq.com/d/d094f793cc3d9cd8

**Comment by Fil** [comment]
And Snyder's chart:

![snyder](https://user-images.githubusercontent.com/7001/59219224-3de75d80-8bc2-11e9-8f41-0faa5c7d071b.png)

**Comment by Fil** [comment]
Nothing forbids having a default angle() — see polyhedral/waterman for example.
